node/src/service/executors.rs

```diff
@@ -26,7 +26,9 @@ pub struct DarwiniaRuntimeExecutor;
 impl NativeExecutionDispatch for DarwiniaRuntimeExecutor {
 	#[cfg(feature = "runtime-benchmarks")]
 	type ExtendHostFunctions = frame_benchmarking::benchmarking::HostFunctions;
-	#[cfg(not(feature = "runtime-benchmarks"))]
+	#[cfg(feature = "evm-tracing")]
```

Comment:
What about you enabling runtime benchmark and evm tracing at the same time?

Reply:
But I think no one would do that.

Reply:
Exactly. evm-tracing is nothing more other than enabling a trace runtime API implementation. And I don't want to add this host function for a normal collator node.